crates/lib/src/action/worker.rs

```diff
@@ -32,17 +32,140 @@ pub async fn worker(
 	events_tx: priority::Sender<Event, Priority>,
 	events: priority::Receiver<Event, Priority>,
 ) -> Result<(), CriticalError> {
-	let mut last = Instant::now();
-	let mut set = Vec::new();
-	let process = ProcessHolder::default();
-	let outcome_gen = OutcomeWorker::newgen();
+	let mut processes: HashMap<SupervisorId, ProcessData> = HashMap::new();
+
+	// Insert the base process from the initial `WorkingData`.
+	let base_process_id = SupervisorId::default();
```

Comment:
Hmm while we're making breaking changes let's also ditch `command` from the working data, so starting a command can only be done from a single place

Reply:
aha, yay! that is a nice change to the api i think yess! having the ability to start commands in just a single place makes it more straight forward to use, yea!